modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/NodeImpl.java

```diff
@@ -3733,6 +3750,11 @@ public Message handleTimeoutNowRequest(final TimeoutNowRequest request, final Rp
                     .build();
             }
 
+            // Keep ordering with current primary.
```

Comment:
Could you please elaborate this? Do you mean that we should use raft options in order to notify primary replica about in-raft safeTime updates?

Reply:
There is a case in raft, then a leader can directly transfer leadership to follower.
For this case it's necessary to include ordering information, because such request ignores leader lease timeout.
Applicable only to timeoutnowrequest.

Reply:
Are there any better options besides using raft options in order to transfer safeTime? If not please add detailed comment similar to

> There is a case in raft, then a leader can directly transfer leadership to follower.
For this case it's necessary to include ordering information, because such request ignores leader lease timeout.
Applicable only to timeoutnowrequest.

Reply:
Added the comment.